app/src/main/java/com/example/playlistmaker/common/presentation/App.kt

```diff
@@ -30,11 +32,6 @@ class App : Application() {
     private fun tyRusskiy() {
```

Comment:
🍏 Рекомендую называть методы на английском языке, а не транслитерировать, так их проще искать по проекту.
Например:
```suggestion
    private fun setRussianLocale() {
```

Reply:
✅